### ISSUE TITLE
Giving Land to Population Group

### ISSUE THREAD

**Issue by Eystein-Thanisch** [cyocum/irish-gen#184]
I have just come across an instance of a king giving land to a population group (LMisc 610, Section 15). However, the ontology for `irishRel:givesLandTo` has the domain "person". Is it possible for it to extend to both people and population groups?

**Comment by Eystein-Thanisch** [comment]
In the same vein, someone also gets fostered to a population group. So can we also extend `irishRel:fosterChildOf` to include population groups?

**Comment by cyocum** [comment]
Not exactly.  There is a way to expand `irishRel:givesLandsTo` by making the domain `foaf:Agent` which would make any reasoner add `foaf:Agent` as a type for URL in the object position. This would be in addition to the declared type.  Another way would be create a new predicate altogether, `irishRel:givesLandToGroup`.  I am slightly leaning towards creating a new predicate.

As for the fosterage one, I would rather create a new predicate, `irishRel:fosteredGroup` and let Tom O'Donnell know we found one for him :P. 

Let me know what you think.

**Comment by Eystein-Thanisch** [comment]
Ok, I am happy with both those ways foward. Shall I change the domain for `irishRel:givesLandTo` or do you want to do it?

As for `irishRel.fosteredGroup`, should that be `irishRel:fosteredToGroup`, with the population group as the object, for clarity?

We should let Tom know. There is quite a lot of context to this particular example that might interest him.

**Comment by cyocum** [comment]
You can go ahead with changing the domain for `irishRel:givesLandsTo`.  It will keep all the changes in one place without the need for rebasing.

As for `irishRel:fosteredGroup`, that sounds find.  You can go ahead and add that as well.

Indeed, would you like to email him or shall I?  If you do, please CC me in as I am interested in it as well.

Thanks!
